MAVProxy/modules/mavproxy_console.py

```diff
@@ -91,11 +91,23 @@ def __init__(self, mpstate):
         # create the main menu
         if mp_util.has_wxpython:
             self.menu = MPMenuTop([])
-            self.add_menu(MPMenuSubMenu('MAVProxy',
-                                        items=[MPMenuItem('Settings', 'Settings', 'menuSettings'),
-                                               MPMenuItem('Show Map', 'Load Map', '# module load map'),
-                                               MPMenuItem('Show HUD', 'Load HUD', '# module load horizon'),
-                                               MPMenuItem('Show Checklist', 'Load Checklist', '# module load checklist')]))
+            mavproxy_items = [
+                MPMenuItem('Settings', 'Settings', 'menuSettings'),
+                MPMenuItem('Show Map', 'Load Map', '# module load map'),
+                MPMenuItem('Show HUD', 'Load HUD', '# module load horizon'),
+                MPMenuItem('Show Checklist', 'Load Checklist', '# module load checklist'),
+            ]
+            
+            # wsproto is not installed by default.
+            # Only add the menu if it's available.
+            try:
+                import wsproto
+            except ImportError:
+                pass
+            else:
+                mavproxy_items.append(MPMenuItem('Start Websocket Server', 'Start Websocket Server', '# output add wsserver:0.0.0.0:56781'))
```

Comment:
Wait.  Why are we adding this if the `ImportError` triggers?

```suggestion
            try:
                import wsproto
                mavproxy_items.append(MPMenuItem('Start Websocket Server', 'Start Websocket Server', '# output add wsserver:0.0.0.0:56781'))
            except ImportError:
                pass
```

... please make sure this code works with/without the Python package present.

Reply:
The video is proof of it working

Reply:
Ah, my bad.  Read "else" as "finally" or whatever...

I don't *really* see the point of `else` here - just seems to spread the code out unnecessarily.

*shrug*